src/components/Individual_Article/List_of_Comments/List_of_Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react"
+import { getArticleComments } from "../../../../api"
+
+const CommentsList = ({articleId}) => {
+    const [articleComments, setArticleComments] = useState([])
+    const [isLoading, setLoading] = useState(true)
+    
+    if(articleId){
```

Comment:
Is this `if statement` necessary? Will a comment list ever not have an articleId passed to it 🤔